dapr_agents/types/llm.py

```diff
@@ -16,13 +16,6 @@ class ElevenLabsClientConfig(BaseModel):
         None, description="API key to authenticate with the ElevenLabs API."
     )
 
-    @field_validator("*", mode="before")
-    @classmethod
-    def none_to_default(cls, v):
-        if v is None:
-            raise PydanticUseDefault()
-        return v
-
```

Comment:
pls note i removed these intentionally to correct an exception i was seeing in the debugger